src/Uno.UI/UI/Xaml/Data/BindingHelper.cs

```diff
@@ -40,12 +40,13 @@ public static DependencyObject GetDependencyObjectForXBind(this DependencyObject
 		public static void ApplyXBind(this DependencyObject instance)
 			=> (instance as IDependencyObjectStoreProvider)?.Store.ApplyCompiledBindings();
 
-		public static void UpdateResourceBindings(this DependencyObject instance)
+		public static void UpdateResourceBindings(this DependencyObject instance) => UpdateResourceBindings(instance, resourceContextProvider: null);
```

Comment:
What happens on older generated code that does not provide the resource context?